.changes/v1.40.0.md

```diff
@@ -0,0 +1,50 @@
+## v1.40.0 - 2023-02-01
+
+### 🎉 Feature
+
+- Add new flags to engine list command. Query flag for partial search, pool-name flag for search by pool name, sort flag for representing data in ascending or descending order, sorted-by flag for specifying field to sort.
+- Add `sort` flag for representing data in ascending or descending order. Add `sorted-by` flag for specifying which field to sort data on the pool list command.
+- Add `sort` flag for representing data in ascending or descending order. Add `sorted-by` flag for specifying which field to sort data on with the role list command.
+- Add `sort` flag to specify ascending or descending order to client search command.
+- Add `sort` flag for representing data in ascending or descending order. Add `sorted-by` flag for specifying which field to sort data on with the policy search command.
+- Add sort, sorted-by flags for representing data in ascending or descending order by some field with the user search command.
+- Add `sort` flag for representing data in ascending or descending order. Add `sorted-by` flag for specifying which field to sort data on with the group search command.
+- Add `query` flag for partial search. Add `limit` flag to specify number of items which will be returned and `cursor` flag to retrieve additional results. Add `sort` flag for representing data in ascending or descending order with the user groups command.
+- Add new `byok update` cli command which allows to change master encryption key for a new user's one.
+
+### 🐛 Bug Fix
+
+- Fix reading cache file for checking cli versions. Save just one using link on update.
```

Comment:
```suggestion
- Fix reading cache file for checking cli versions. On update only one file will be used now.
```
Reworded for clarity.

Reply:
I've changed it with my own words. Please take a look